server/controller/src/main/java/ai/starwhale/mlops/domain/job/cache/HotJobHolder.java

```diff
@@ -27,13 +27,17 @@
  */
 public interface HotJobHolder {
 
-    void adopt(Job job);
+    void add(Job job);
```

Comment:
what is the benefit of renaming this method

Reply:
It's just a bit hard to see the effect at a glance